rasa_nlu/tokenizers/jieba_tokenizer.py

```diff
@@ -56,6 +60,11 @@ def process(self, message, **kwargs):
     def tokenize(self, text):
         # type: (Text) -> List[Token]
         import jieba
+
+        dictionary_path = self.component_config['dictionary_path']
```

Comment:
`.get('dictionary_path')`

Reply:
Nice one. Make the code more robust. I will update the code very soon.